code/square_root_decomposition/src/mos_algorithm/sqrtdecomposition.py

```diff
@@ -0,0 +1,66 @@
+# PROBLEM STATEMENT-> You are given an array A of size N and Q queries.(All indexes are 1 based).
```

Comment:
Reformat this file using black.

Reply:
Sir in my last PR , Firstly I have copied my code pasted in the left side of the black then copied from the right side and saved into a file . Then I pushed that file in gituhub. 
Is there any other procedure to reformat it??

Reply:
No you are correct. My mistake I was looking at the wrong file in the Travis output.

Reply:
Thanks a lot for merging ...